firmware/ci.c

```diff
@@ -231,6 +231,7 @@ static char *readstr(void)
 	char c[2];
 	static char s[128]; // Needs to fit a full mode line ~100 chars.
 	static int ptr = 0;
+	static char skip = 0;
```

Comment:
Should this technically be `'\0'` ?

Reply:
Yes, I was just following the style.  I can update other places they are assigning char to `0` to `'\0'`